test/sql/storage/compression/bitpacking/bitpacking_types.test_slow

```diff
@@ -13,7 +13,7 @@ foreach bitpacking_mode delta_for for constant_delta constant
 statement ok
 PRAGMA force_bitpacking_mode='${bitpacking_mode}'
 
-foreach type <numeric> hugeint decimal(4,1) decimal(8,1) decimal(12,1) decimal(18,1)
```

Comment:
What is the reason `hugeint` was removed here?

Reply:
`hugeint` is already part of `numeric`.